run.sh

```diff
@@ -255,15 +275,22 @@ check_themes() {
     done
   fi
 
-  [[ "${#to_install}" -gt 0 ]] && wp theme install --allow-root "${to_install[@]}" | tail -n 1 |& _colorize
+  for key in "${to_install[@]}"; do
```

Comment:
We used to do it this way (looping through each plugin individually), but we opted to lump them all together in the same command to speed up the build time.

Full confession: When we made this decision, we only assumed that wp-cli installs plugins concurrently. I've still never looked into it to find out.

Can you check on this and let me know? If wp-cli _does_ do the installs concurrently, can we try to go that route?

Reply:
Plugin upgrades are **not** concurrent

If you follow the call stack: [wp plugin install](https://github.com/wp-cli/extension-command/blob/master/src/Plugin_Command.php#L707 ) calls into the [base class](https://github.com/wp-cli/extension-command/blob/master/src/WP_CLI/CommandWithUpgrade.php#L125) and does a [foreach](https://github.com/wp-cli/extension-command/blob/master/src/WP_CLI/CommandWithUpgrade.php#L128) slug

Reply:
I can still change if desired, just figured I would give you the first answer.